lib/rules/jsx-no-bind.js

```diff
@@ -168,13 +169,15 @@ module.exports = {
         if (!node.init) {
           return;
         }
-        const blockAncestors = getBlockStatementAncestors(node);
+        const blockAncestors = getBlockStatementAncestors();
         const variableViolationType = getNodeViolationType(node.init);
 
         if (
           blockAncestors.length > 0
           && variableViolationType
+          && node.parent.type === 'VariableDeclaration'
```

Comment:
we're already inside VariableDeclarator, this shouldn't be needed

Reply:
`Property 'kind' does not exist on type 'Node'.`
This error occurs.

Reply:
if this is just for narrowing, we can use type casts to tell TS what kind of thing it is.

Reply:
In my understanding, since it's a JavaScript file, I don't think type assertions can be used.

Reply:
they can, by wrapping the value in parens and preceding it with a `@type comment.

Reply:
I didn't know that method! Thank you!  Fixed in e2c3497 .